tests/v2/validation/observability/rbac/stackstate_rbac_test.go

```diff
@@ -0,0 +1,257 @@
+package observability
```

Comment:
I'd prefer a different name, but I see why you picked it. I feel like most will have to look at the filenames in this folder / at the tests themselves to know this is exclusively stackstate though.

Reply:
StackState is officially called SUSE Observability (as referenced in https://docs.stackstate.com/get-started/k8s-getting-started, so I named the package accordingly to reflect its official name. 

might be a question for @rancher-max should we name the package stackstate or observability.